samples/echo/build.gradle

```diff
@@ -0,0 +1,39 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 27
+
+    defaultConfig {
+        applicationId 'com.google.sample.oboe.echo'
+        minSdkVersion 24
```

Comment:
Oboe samples should all have a minSdk of 16. They should also be tested on a device or emulator running API 16.

Reply:
yeah got this part: some portion of the UI code needs to be changed and done: will get a device to do a real test: failed to create recording string on api-16 + emulator, could be some bugs